### DISCUSSION TITLE
An attribute "api_version" value format in "responses.info.api.Information" component "/info" endpoint

### DISCUSSION THREAD

**Discussion by asafieva** [CBIIT/ccdi-federation-api#103] [API Design Discussion]
Do we want the same attribute "api_version" value format in  "responses.info.api.Information" component "/info" endpoint? If yes, what is a format preference? as "api_version": "v1.0.0"? Is it OK to have this value formatted differently in different Federation Nodes? 
```"api": {
        "api_version": "v1.0.0",
        "documentation_url": <...>
    }

**Comment by e-t-k** [comment]
Hi Natalia,
`api_version` refers to the version of the interface, which is shared between everyone, so I think we should ask Federation Nodes to all use the same format, and that format should also be what this repository uses.
So, for version 1, since the repository is tagged [v1.0.0](https://github.com/CBIIT/ccdi-federation-api/releases/tag/v1.0.0 ), I think all nodes should also use `"v1.0.0."`. 
If we want to change the format we should do that starting with the next release.
